tests/test.js

```diff
@@ -266,6 +266,25 @@ mpmodule("mixpanel.track");
         });
     });
 
+    asyncTest("check no property name aliasing occurs during minify", 1, function() {
```

Comment:
Added a new test for token override here: bd749ab and left this test (which will detect 'ad' and 'ab' and other minified name bugs)